src/group_theory/nilpotent.lean

```diff
@@ -395,3 +397,16 @@ begin
     (le_trans ((map_eq_bot_iff _).mp _) hf1)⟩,
   exact eq_bot_iff.mpr (hn ▸ (lower_central_series.map f n)),
 end
+
+lemma derived_in_lower_central (n : ℕ) : derived_series G n ≤ lower_central_series G n :=
+by { induction n with i ih, simp, exact (general_commutator_mono ih (by simp)) }
```

Comment:
```suggestion
by { induction n with i ih, { simp }, { apply general_commutator_mono ih, simp } }
```
I don't know how strict our style guidelines are about this, but the proof becomes easier to read when you separate subgoals using brackets. And it's usually good to avoid nested tactuc blocks (i.e. writing `by ...` *inside* a tactic block).

Reply:
The `exact general_commutator_mono ih (by simp)` is fine by me (I'm a bit `apply`-phobic),
```suggestion
by { induction n with i ih, { simp }, { exact general_commutator_mono ih (by simp) } }
```
but I'm happy with what David suggested.